python/aswfdocker/data/versions.yaml

```diff
@@ -869,6 +871,8 @@ groups:
         - osl
         - openvdb
         - vfxall
+      review:
+        - openrv
```

Comment:
I wasn't really sure about that part since OpenRV is not a library like the other one.